OSIsoft_Demo/src/OSIsoftServer.java

```diff
@@ -45,6 +45,74 @@ public OSIsoftServer(String ip, String login, String webID) {
       postHeaders = "Authorization=Basic " + authCredentials + "&Content-Type=application/json";
    }
    
+   // Looks up and sets a tags PI Point web id.  If the tag does not exist
+   // in the data server a PI Point is created for that tag.
+   public void setTagWebId(Tag tag) throws JSONException {
+      
+      //HTTPS responses are stored in this file
+      String responseFilename = "/usr/response.txt";

```

Comment:
we have to store the response into a file vs. a variable?

Reply:
Unfortunately yes.

Reply:
A comment about how we can't retrieve the values into a variable would make it more clear imo about the current limitation of requesthttpx.